CMakeLists.txt

```diff
@@ -265,7 +265,7 @@ if (NOT MSVC)
                       -Wcast-qual
                       -Woverloaded-virtual
                       -Wignored-qualifiers)
-  if (CMAKE_COMPILER_IS_GNUCC AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 5.1)
+  if (CMAKE_CXX_COMPILER_IS_GNUCC AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 5.1)
```

Comment:
Turns out this was broken if you're unintentionally using clang for c++ and gcc for c like I was.